### PR TITLE
Docstrings for v0.3

### PR DESCRIPTION
**Pull request by Wikunia**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [ ] Did I make sure to only change the part of the file where I introduced a new change/feature?
  - I've realized that we missed to bound `Hungarian`...
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**How did you address these issues with this PR? What methods did you use?**
Docstrings for the `anim_` functions. I've added a `# Possibilities` to show the different argument possibilities this time. Hope that makes sense.
I've also added a docstring for `act!` as that was missing. Maybe it can be simplified as the possibilities are quite simple.

### CODE REVIEW

File: src/structs/Transitions.jl [hunk 1]
```diff
@@ -19,9 +36,45 @@ struct Rotation{T<:Real} <: AbstractTransition
...
+    anim_rotate_around
+
+Animate the rotation of the attached object (see [`act!`](@ref)) around a point.
+Similiar function: [`anim_rotate`](@ref) to rotate or spin an object
+
+# Example
+```julia
+Background(1:100, ground)
+obj = Object((args...) -> rect(O, 50, 50, :fill), Point(100, 0))
+act!(obj, Action(1:50, anim_rotate_around(2π)))
```
Comment (TheCedarPrince) [inline src/structs/Transitions.jl:70]: Is this option defined in the Options section here? I don't clearly see it.

File: src/structs/Transitions.jl [hunk 2]
```diff
@@ -37,5 +90,22 @@ Scaling(fs::Tuple, ts) = Scaling(Scale(fs...), ts)
...
+# Example
+```julia
+Background(1:100, ground)
+obj = Object((args...) -> rect(O, 50, 50, :fill), Point(100, 0))
+act!(obj, Action(1:50, anim_scale(1.5)))
+```
+
+# Options
+- `anim_scale(ts)` scales from the current scale to `ts`.
+- `anim_scale(fs, ts)` scales from `fs` to `ts`.
```
Comment (TheCedarPrince) [inline src/structs/Transitions.jl:108]: I get that `ts` and `fs` represent "to scale" and "from scale" but could you add a type definition of whether they accept Floats, Ints, Reals? Thanks!

**Review by TheCedarPrince — CHANGES_REQUESTED** [review]
I made a few grammar changes and typo fixes as well as cleaned up some of the docstrings. Just a few comments to resolve and then I think it is great.

File: src/structs/Transitions.jl (hunk 2 above)
Comment (Wikunia) [reply src/structs/Transitions.jl:108]: There are quite a lot of different ways like Symbol, Number and object

File: src/structs/Transitions.jl (hunk 1 above)
Comment (Wikunia) [reply src/structs/Transitions.jl:70]: Which option?

Comment (Wikunia) [reply src/structs/Transitions.jl:70]: I forgot defining a position there. Copy and paste error :smile:  
Thanks

File: src/structs/Transitions.jl (hunk 2 above)
Comment (Wikunia) [reply src/structs/Transitions.jl:108]: Maybe it's clearer now. I didn't document the symbol there as it's more for internal usage.

**Review by TheCedarPrince — APPROVED** [review]
LGTM! Feel free to merge.
